UnicornRender/video/include/unicorn/video/Primitives.hpp

```diff
@@ -22,15 +22,17 @@ class Primitives
     *  @brief Fills given mesh with cube data
     *
     *  @param[out] mesh    geometry mesh
+    *  @return pointer to updated mesh
     */
-    UNICORN_EXPORT static void Cube(Mesh& mesh);
+    UNICORN_EXPORT static Mesh* Cube(Mesh& mesh);
```

Comment:
for streamlining calls I would suggest to return the same type you take - reference in this case